### PR TITLE
Mobile fixes

### PR DESCRIPTION
**Pull request by mktcode**
Proposal page:
- put author/dropdowns in new line
- reduced text size/line height
- reduced paddings in votes block

Proposal list:
- reduced text size/line height
- reduced paddings

Timeline:
- removed borders
- (no other adjustments for now because I will try to merge those two components TimelineProposal.vue and TimelineProposalPreview.vue)

Looks good to me now all the way down to 360px width, which is still the most common according to [some statistic](https://gs.statcounter.com/screen-resolution-stats/mobile/worldwide) I found.

Share/dropdown menus would look better if they were on the same line as proposal state and space but that's a bit tricky if author is meant to be in the middle on large screens. Some other day...

image

image

### CODE REVIEW

**Comment by bonustrack** [comment]
I think we should use the full width of the page on mobile, currently width look way too small just enough for 5 words. I would make the sidebar horizontal or colapsed by default

File: src/views/Timeline.vue [hunk 1]
```diff
@@ -176,7 +176,7 @@ function selectState(e) {
           </template>
         </BaseDropdown>
       </div>
-      <div class="md:rounded-lg bg-skin-block-bg border">
+      <div class="md:rounded-lg bg-skin-block-bg md:border">
```
Comment (samuveth) [inline src/views/Timeline.vue:179]: Timeline now looks like this on darkmode, not sure why
image

**Review by samuveth — APPROVED** [review]
Otherwise looks great!

File: src/views/Timeline.vue (hunk 1 above)
Comment (mktcode) [reply src/views/Timeline.vue]: Thanks. Fixed that and left a comment in styles.scss.

**Pull request merged**